Nanachi missions and variants (#4685)
endless-sky/endless-sky: data/hai missions.txt

	on visit
		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for her to arrive in this star system.`
	on complete
		conversation
			`You help the family gather all their luggage and bid them farewell, but right as they disappear into a hallway Nanachi lets out a heavy sigh. "I bit off more than I could chew. Those kids kept wanting to play with the ship's controls, and one of them tried to go up the air vent in a game of hide-and-seek. I couldn't get any rest since I had to keep making sure they weren't about to dump their own bags into the void by accident."`
			`	"Do you want to get some sleep before trying to find another mission?" you ask, hoping she won't try to overwork herself. "Yes, I could really use that. Sorry to make you wait again captain <last>. I'll rest a bit, see if I can't find a tourist spot nearby to relax, and meet up with you the in spaceport after a while."`

mission "Nanachi 3"
	name "Deliver spices with Nanachi"
	description "Accompany Nanachi to Alta Hai to pick up some Quarg spices for a festival in <planet>."
	source "Hai-home"
	stopover "Alta Hai"
	destination "Allhome"

> 	deadline 11

Reviewer: This deadline also doesn't allow any wiggle room whatsoever. Add a day or three.

			`	They tell Nanachi that they won't be going along, they already have a transport to Allhome, and will stay here a bit longer to try and advertise the event.`
				accept
	on stopover
		dialog
			`A spaceport worker contacted Nanachi shortly before you two landed and made sure the spices were ready to be loaded in her ship. You help the workers and Nanachi make sure everything is in order, and prepare for your trip to Allhome.`
	npc accompany save
		personality escort timid
		government "Hai"
		ship "Heavy Shuttle" "Acorn Chaser"
	on visit
		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system.`
	on complete
		conversation
			`Allhome's docking area shows much more traffic than usual. Many captains here are helping workers open cargo containers full of food, clothes, books, movies and even some mounted animals from human space. You finally meet up with the festival's organizers, who seem very excited to have acquired the exotic condiments, and have invited you two to a bar in the spaceport for a drink.`

mission "Nanachi 4"
	name "Protect Nanachi"

> 	description "Bring Nanachi's ship to Prime, keeping her safe from a pirate fleet that attacked a convoy heading to Hai space, then head back to <planet>."

Reviewer: ```suggestion
	description "Bring Nanachi's ship to <stopovers>, keeping her safe from a pirate fleet that attacked a convoy heading to Hai space, then head back to <destination>."
```
Needs to mention the deadline somewhere.

Author: Changed, and included the date/deadline.

`	Vanessa grabs ahold of Nanachi's arm before she can run off to the spaceport. "Just remember, if you can't get back here by <day>, don't worry about it; we'll understand. I'm sure we can come up with some alternative to have the festival without our main attractions."`

Author: Changed to include deadline, as well as dialog mentioning it.

`	Vanessa grabs ahold of Nanachi's arm before she can run off to the spaceport. "Just remember, if you can't get back here by <day>, don't worry about it; we'll understand. I'm sure we can come up with some alternative to have the festival without our main attractions."`

	source "Allhome"
	stopover "Prime"

> 	deadline 14

Reviewer: Allhome to Prime and back is exactly 14 days with no mistakes. Usually it's good to give some wiggle room. Perhaps 3 weeks instead of 2, or 16/17 days? (Dialog in the conversation would need adjusted accordingly.)

	to offer
		has "Nanachi 3: done"
	on offer
		conversation

> 			`The bar is just as busy as the docks. Many captains here must have transported something for the festival. "Captain <last>, captain Nanachi!" shouts Narancia, beckoning the two of you to join her and the others at their table. You learn that they were inspired by the Hai's own March festival, and wanted to bring small parts of human culture to Hai space.`

Reviewer: ```suggestion
			`The bar is just as busy as the docks. Many captains here must have transported something for the festival. "Captain <last>, Captain Nanachi!" shouts Narancia, beckoning the two of you to join her and the others at their table. You learn that they were inspired by the Hai's own March festival, and wanted to bring small parts of human culture to Hai space.`
```

			`	"They can't go to our planets and experience it for themselves yet, so we thought about bringing it to them!" Vanessa says.`

> 			`	"It was very hard to get enough sponsors for this, but after years of hard work we even managed to have famous art pieces be brought here, some of which date back to the European renaissance!" Xavier adds.`

Reviewer: ```suggestion
			`	"It was very hard to get enough sponsors for this, but after years of hard work we even managed to have famous art pieces be brought here, some of which date back to the European Renaissance!" Xavier adds.`
```
I'm curious as to what they got their hands on. 🤔 I imagine Michelangelo's David surrounded by tables of snacks.

			`	Afterwards the conversation topic shifts to you as a captain, as you describe your journey after buying your first ship, and how you met Nanachi. You're about to order another drink for the group when a grizzled old man barges into the bar, and starts shouting: "The art convoy has been attacked! The freighter managed to escape but they're stranded on Prime now, and the pirates are still out there, patrolling the route to the wormhole!"`
			`	Xavier stands up with a shocked expression. "How? We had a Leviathan escorting them!"`
			`	After catching his breath, the man at the door responds. "The pirates must have been stealing Hai tech from ships going back home. The freighter's captain said they shredded the escort!" Chaos spreads amongst the other captains, and the group sitting with you is in anguish.`
			choice
				`	"Are those art pieces really that valuable?"`
				`	"Can't you just postpone the festival until the pirates are taken care of?"`

> 			`	Xavier collapses back to the table, holding his head in both hands. "Those paintings and sculptures were our main attraction! They're worth tens of millions! It'll take a month before the Navy ever notices the pirates there and makes it safe for the convoy to come here. The art exhibition is scheduled two weeks from now!"`

Reviewer: >They're worth tens of millions!

Depending on the specific pieces involved and given the age mentioned, they may be worth billions, or even be priceless. The player themselves needs a net worth of 2 billion to even get to this point, so perhaps give more emphasis to the importance of the art. Maybe even don't mention that they were the main attraction, with Xavier being more concerned about the historical value of the art.

(At the very least, change "tens of millions" to "billions". The rest of the suggestion could be undesired depending on how we want Xavier and Vanessa to be characterized.)

			`	As they try to figure out what to do, Nanachi speaks up. "I'll go there. I pilot a human ship and if we just need to transfer cargo from one ship to the other, I could get out of there quickly. Nobody would notice that a Hai ever went there."`
			`	"Are you insane?" Vanessa says, "If they managed to take down that Leviathan, you wouldn't last a second! Besides, our convoy was extremely fast. If even they were caught your chances of outrunning the pirates are next to nothing!"`

> 			`	"Captain <last>," Nanachi turns to you. "I know you already helped me way more than anyone would ever be willing to, and I'm really grateful for that, but would you be willing to help me one last time, to get to that planet, and travel back here?" While it's certain that for a pirate fleet to take down a Leviathan so easily they must be incredibly powerful, and this could be extremely dangerous, you're sure that if Nanachi tries to get to Prime by herself she'll just get herself killed.`

Reviewer: ```suggestion
			`	"Captain <last>," Nanachi turns to you. "I know you already helped me way more than anyone would ever be willing to, and I'm really grateful for that, but would you be willing to help me one last time, to get to that planet, and travel back here?" While it's certain that for a pirate fleet to take down a Leviathan so easily they must be incredibly powerful, and this could be extremely dangerous. You're sure that if Nanachi tries to get to Prime by herself she'll just get herself killed.`
```

				accept
	npc
		personality nemesis waiting
		government "Pirate"
		system "Betelgeuse"
		ship "Marauder Firebird (Hai)" "Nut King Call"
		ship "Marauder Splinter (Hai)" "The Nutcracker"
		ship "Marauder Quicksilver (Hai)" "Hazelnut's Bane"
		ship "Marauder Bounder (Hai)" "Rodent Hunter"
	npc accompany save
		personality escort timid
		government "Hai"
		ship "Heavy Shuttle" "Acorn Chaser"
	on visit
		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system.`
	on stopover
		conversation
			`The convoy's captain messaged you as you entered the system and told you to land on a remote, hidden area of Prime, where they have already prepared for all the cargo to be transferred. They refuel your ships and urge you two to go back. "Captain Nanachi can stay hidden with us here if you need supplies or anything from the outfitter, but please don't take too long."`
	on complete
		event "Nanachi: payment" 180
		conversation

> 			`Landing back on Allhome, the festival group gives you and Nanachi a warm hug, "We sold tens of thousands of tickets for the festival by promising to host a great art exhibition with pieces from all over human space. Without them it'd be a complete failure, and we'd be in serious trouble with our sponsors." Narancia explains.`

Reviewer: This would need to be changed to move the focus to the value of the art instead of the sake of the festival if you make the suggested change above.

Author: The value of the art shouldn't matter; you to go see the art because it's interesting, not because it is expensive.

> 			`	As the cargo is rushed to the exhibition site, Nanachi shows you the chip she got as payment, and the balance of her bank account: this last payment allowed her to fully pay off her loan. "Captain," she speaks with some difficulty, and tries to hide her face, as if to wipe away some tears. "I'll never be able to thank you properly, if not for you I'd still be lost in that spaceport, and would eventually have to go back to my mining job. I'll work hard now; I'll save up and pay you for everything!" She hugs you and bows to you in a way you'd see in movies, as if trying to imitate the gesture. You two finally say goodbye, and head back to your ships.`

Reviewer: Damn, I wish I could've paid off my loan in 4 missions.

Author: You can!  Just capture some poor pirate ships and sell them off. :)

event "Nanachi: payment"

mission "Nanachi: Payment"
	landing
	source
		government "Hai"
		not planet "Darkwaste"
	to offer
		has "event: Nanachi: payment"
	on offer
		payment 700000
		event "Nanachi: Update" 720
		conversation

> 			`Shortly after you land, you get a message from Nanachi, the young hai pilot who you helped start out. "Dear captain <last>, I hope you're doing well! As I promised, ever since we parted ways, I've been working towards getting enough money to pay you for being so kind to help me. I've sent you <payment>. I'm deeply grateful to you for helping make my dream come true."`

Reviewer: ```suggestion
			`Shortly after you land, you get a message from Nanachi, the young Hai pilot who you helped start out. "Dear Captain <last>, I hope you're doing well! As I promised, ever since we parted ways, I've been working towards getting enough money to pay you for being so kind to help me. I've sent you <payment>. I'm deeply grateful to you for helping make my dream come true."`
```

				decline

event "Nanachi: Update"

mission "Nanachi Meeting"
	landing
	source
		government "Hai"
		not planet "Darkwaste"
	to offer
		has "event: Nanachi: Update"
		random > 30
	on offer
		conversation
			`After you leave your ship and start making your way through the docking area, you notice a Hai running towards you and waving. As they get closer, you realize that it's Nanachi, the Hai pilot who you helped out years ago.`
			`	"I was just about to go back to my crew when I saw you coming out of your ship. I know a nice snack bar here, let's go there!" Before you can even say hello, she's dragged you to the bar, and ordered some drinks for the two of you. She tells you about her last few years as a captain. She has long since sold her old ship, and now pilots a Geocoris, transporting large amounts of cargo across Hai space. You also tell her about how your life has been ever since you last met, of your adventures and how far you've come. You keep swapping stories for hours, until the bar is ready to close.`
			`	You thank her for the drinks, and get ready to say your goodbyes.`

> 			`	"Oh wait, before you go captain, let me give you something," She says, right as you're about to head to your ship. She grabs a small pendant from her pocket. "Back home in Giverstone, when I had some free time from the mining works, I took a hobby of sculpting some small stones into ship miniatures. I thought it'd be nice to make this for you in case I saw you again." She gives you the pendant, which has been sculpted into a Hai Shield Beetle. It is extremely well made, with different surface textures representing the different parts of the Shield Beetle. "It's just a good-luck charm I thought you'd like. Thanks again captain!" You two part ways, and head back to your ships.`

Reviewer: ```suggestion
			`	"Oh wait, before you go Captain, let me give you something," she says, right as you're about to head to your ship. She grabs a small pendant from her pocket. "Back home in Giverstone, when I had some free time from the mining works, I took a hobby of sculpting some small stones into ship miniatures. I thought it'd be nice to make this for you in case I saw you again." She gives you the pendant, which has been sculpted into a Hai Shield Beetle. It is extremely well made, with different surface textures representing the different parts of the Shield Beetle. "It's just a good-luck charm I thought you'd like. Thanks again Captain!" You two part ways, and head back to your ships.`
```

mission "Nanachi 1"
	name "Escort Nanachi to <planet>"
	description "Help Nanachi, a Hai who just bought her first ship, make a simple delivery to Cloudfire."
	minor
	source "Giverstone"
	destination "Cloudfire"
	deadline
	to offer
		"net worth" > 20000000
		"combat rating" > 1200
		year > 3018
	on offer
		conversation
			`As you are making your way through the spaceport you notice a worried-looking Hai staring at the local job board and back at some sort of document they are holding.`
			choice
				`	(Approach the Hai.)`
				`	(Leave them alone.)`
					decline
			`	The Hai is a bit startled, but quickly calms down. "Sorry, you caught me by surprise. I've just bought my first ship and need to find some jobs to do, but nobody I've talked to wants to trust a beginner captain. I just don't know what to do anymore, I've been working for so long to qualify for this loan, but I'm running out of time and I'll never be able to pay it like this."`

> 			`	Turns out the documents she is holding are the bank papers for the loan, somewhat similar to the one you took when you bought your first ship. You've come a long way since your days as a beginner, and know how hard it is to be starting out as a starship captain, you could probably convince someone here to offer a simple cargo job to this young Hai.`

Author: I get the feeling that the part you want to break off is the dependent clause of the prior statement.  You're only thinking of helping because you know the issues when you're starting out.
My gut says either leave it, or change the comma to a semi-colon.

			choice
				`	(Offer to help her find some jobs.)`

> 				`	(Wish her luck and leave.)`

Author: Changed to have a second chance.  Those two second-chance choices are `(action)`.

The two choices are now
"If you're interested, I could try to help you find a job."
"I've been there before; trust me, it gets better. I wish you the best of luck on finding a job."

Second one feels weird though...  I mean, you're essentially going up to someone, and saying "hey, I can't help, but good luck.".  Seems a bit... strange.

> 				`	(Wish her luck and leave.)`

Author: Changed.  Just feels a bit weird since you're going up to a stranger, offering help...

					decline

> 			`	Her ears twitch a bit, and her previously crestfallen eyes light up as she nearly jumps in joy at your offer. "Thank you so much captain! My name is Nanachi, I promise I won't cause any trouble at all, just a few quick jobs and I'm sure I'll be fine by myself!"`

Author: Change done elsewhere.  (How did I miss this from before...)

> 			`	Her ears twitch a bit, and her previously crestfallen eyes light up as she nearly jumps in joy at your offer. "Thank you so much captain! My name is Nanachi, I promise I won't cause any trouble at all, just a few quick jobs and I'm sure I'll be fine by myself!"`

Author: Changed elsewhere.

			`	You introduce yourself to Nanachi and shake her paw. "Can you show me your ship so I can look for jobs it's best suited for?" you ask.`
			`	She hesitates a bit, but leads you to her ship. You were expecting a small Hai ship, but to your surprise she shows you a rather beat-up Heavy Shuttle docked near your own ship.`
			`	"You took a loan to buy a human ship?" you ask.`

> 			`	She looks at the ground, as if ashamed of it. "It'd take way too long for me to qualify for a loan for even an Aphid, but I recently met a human merchant who wanted to sell his old ship; it was difficult but I managed to convince him to sell it to me instead of going to the shipyard office. I paid him its full price, but even for this one I had to take out the biggest loan I could. Its cargo hold isn't very impressive but it's got more bunks than our smallest ships, if that helps."`

Author: Not sure why you're breaking up that first `but` but not capitalizing it...
Will add the comma before the second `but`.

> 			`	She looks at the ground, as if ashamed of it. "It'd take way too long for me to qualify for a loan for even an Aphid, but I recently met a human merchant who wanted to sell his old ship; it was difficult but I managed to convince him to sell it to me instead of going to the shipyard office. I paid him its full price, but even for this one I had to take out the biggest loan I could. Its cargo hold isn't very impressive but it's got more bunks than our smallest ships, if that helps."`

Reviewer: Yeah, meant to add a comma, not a period.

				accept
	npc accompany save
		personality escort timid
		government "Hai"
		ship "Heavy Shuttle" "Acorn Chaser"
	on visit
		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for her to arrive in this star system.`
	on complete
		conversation
			`Shortly after her ship lands, you meet up with Nanachi, who is doing her best to hide her excitement at being on another planet for the first time, and is gawking at some Hai military ships in particular. "So, how did you like your first time doing a hyperspace jump?" you ask, to try and snap her out of it.`

> 			`	"Oh um, I don't think I'll get used to the feeling anytime soon. I threw up a lot so I'm glad this wasn't a job with passengers. I've always dreamt of being on other worlds, exploring and learning everything about them. I'm glad I'm finally able to do it."`

Author: On one hand, that could be expanded into a little funny scene; Nanachi coming out of the ship wet, mentioning how she wanted to clean herself but got caught up in piloting and showering at the same time, how she caused a minor overflow due to Hai hair shedding vs human shedding and the differences in construction and all that...
On the other, I would expect that barf bags exist, and that even a shuttle should have basic bathroom facilities, including some way to dry yourself off.  These are ships that are expected to be out in space for about a week alone (ok, maybe half a week), so basic amenities like that should be included.

			`	You help her get the crates out of the cargo hold and contact the client. Nanachi's share of the payment is small, as the client transfers most of the money to the merchant she got the job from, but that doesn't make her any less excited about getting her first paycheck.`
			`	"So, want to go to the spaceport and try to get a job on your own? I'll help you again in case you can't do it yet." you say.`

> 			`	"Just give me an hour or two, back home our shipyard is kind of small, and I've never really gotten a chance to see a Shield Beetle up close, and there are many undergoing repairs here, I'll just go get a better look at them, I swear I'll be quick."`

Author: She's excited about seeing the Shield Beetles, just finishing her first job... so she's probably not thinking too much about breaking up her words into sentences..  Think of it as the result of writing what she's actually saying, vs how we'd want it to come out.

As for "hour or two", that's just an idiom/guess.

> 			`	"Just give me an hour or two, back home our shipyard is kind of small, and I've never really gotten a chance to see a Shield Beetle up close, and there are many undergoing repairs here, I'll just go get a better look at them, I swear I'll be quick."`

Reviewer: Same here.

> 			`	You two agree to meet in the spaceport in a few hours, maybe this time she will do fine on her own.`

Author: I feel like this should be a semi-colon.  It's not enough for its own sentence, but different enough from the initial one.

mission "Nanachi 2"
	name "Escort Nanachi to <planet>"
	description "Bring Nanachi and her passengers to <destination>."
	source "Cloudfire"
	destination "Hai-home"
	to offer
		has "Nanachi 1: done"
	on offer
		conversation

> 			`You meet up with Nanachi in the spaceport, give her some tips to try and persuade someone into hiring her, and tell her you'll be nearby if she still has trouble getting a job. You watch her go around the spaceport and approach some groups of humans and even other Hai to no avail, maybe since this is a military world, they aren't too keen on helping others as in the rest of their territory. She manages to grab the attention of a Hai family leaving a restaurant carrying way too much luggage, most likely tourists looking for a ride back home. Everything seems to be going well until the parents start to look worried. It seems as if they might go away if you don't step in.`

Author: I'd say the helping goes both ways; she's a relatively new pilot, on a military world...  Anyone agreeing to go with her is going to have to accept some increased risk, and possibly have to help out (more along the lines of paperwork and back-end stuff, not fighting...).

> 			`You meet up with Nanachi in the spaceport, give her some tips to try and persuade someone into hiring her, and tell her you'll be nearby if she still has trouble getting a job. You watch her go around the spaceport and approach some groups of humans and even other Hai to no avail, maybe since this is a military world, they aren't too keen on helping others as in the rest of their territory. She manages to grab the attention of a Hai family leaving a restaurant carrying way too much luggage, most likely tourists looking for a ride back home. Everything seems to be going well until the parents start to look worried. It seems as if they might go away if you don't step in.`

Author: The suggestion has been added in another commit.

As for the help, I'm going with what I said before.

			`	You walk over to the group to help her, and get a better look at the family: a Hai couple carrying a baby, and three older children who aren't doing much to help their father carry their ludicrous number of bags.`
			choice
				`	"Hey, what's going on?"`
				`	"Can I be of assistance?"`

> 			`	You introduce yourself and the parents tell you that they've been on vacation here and need transport back to Hai-home. "Don't take this the wrong way, but captain Nanachi told us her ship is one of your human ships, a Heavy Shuttle was it? It has enough bunks for us but our sons already caused a ruckus on the way here in a Hai ship, we're worried that since they aren't used to the accommodations, they might end up giving her too much trouble."`

Author: She's in charge of her own ship, which is, from what I can tell, the traditional requirement of being called Captain.  Being good at the job isn't a requirement...  (I mean, look at the various maritime wrecks caused by bad Captains...  Costa Concodia comes to mind.)

Will change to `Captain`.

> 			`	You reassure them that the trip will be very quick, and that the ship's interior is somewhat similar to that of Hai ships, until they finally agree on going with her. They settle on the pay and you two help the Hai get their baggage to Nanachi's ship. When the family is focused on a conversation, you quietly ask Nanachi what was the issue she was having with the other people she encountered.`

Author: Maybe shift from `going` to `traveling`?
`and the two of you` instead of `and you two`.

As for the issue, the player only knows that Nanachi was the common thread; therefore asking what issue she had seems reasonable.  If it was going to be changed, it would have to recognize that each of the people she talked to could have had different issues, so `what the issues were with the other people` or so.

> 			`	You reassure them that the trip will be very quick, and that the ship's interior is somewhat similar to that of Hai ships, until they finally agree on going with her. They settle on the pay and you two help the Hai get their baggage to Nanachi's ship. When the family is focused on a conversation, you quietly ask Nanachi what was the issue she was having with the other people she encountered.`

Author: This was changed to "traveling" and "the two of you".  I feel the change from Going to Traveling is sufficient, as you're already convincing them with the start of the sentence.

				accept
	npc accompany save
		personality escort timid
		government "Hai"
		ship "Heavy Shuttle" "Acorn Chaser"
	on visit
		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for her to arrive in this star system.`
	on complete
		conversation
			`You help the family gather all their luggage and bid them farewell, but right as they disappear into a hallway Nanachi lets out a heavy sigh. "I bit off more than I could chew. Those kids kept wanting to play with the ship's controls, and one of them tried to go up the air vent in a game of hide-and-seek. I couldn't get any rest since I had to keep making sure they weren't about to dump their own bags into the void by accident."`
			`	"Do you want to get some sleep before trying to find another mission?" you ask, hoping she won't try to overwork herself. "Yes, I could really use that. Sorry to make you wait again captain <last>. I'll rest a bit, see if I can't find a tourist spot nearby to relax, and meet up with you the in spaceport after a while."`

mission "Nanachi 3"
	name "Deliver spices with Nanachi"

> 	description "Accompany Nanachi to Alta Hai to pick up some Quarg spices for a festival in <planet>."

Author: Is mentioned now in both the description, and by Nanachi herself in dialog.

	source "Hai-home"
	stopover "Alta Hai"
	destination "Allhome"
	deadline 11
	to offer
		has "Nanachi 2: done"
	on offer
		conversation

> 			`You send a message to Nanachi and you two meet up in the spaceport. You're still walking toward the job board, when she asks: "Back home I heard stories about how humans have a much larger population than the Hai, and that their home planet has even more people than Hai-home, is it true that there are many other human worlds with spaceports this large?" You name her a few human planets that are extremely populous, and how even some planets with fewer people than this world have incredibly large spaceports.`

Author: You're assuming a uniform distribution of population across systems though.  We know a large chunk of those Human systems barely reach a few million people.  The South, Rim, Dirt Belt... aside from a few specific worlds, I wouldn't put their population in the 100's of millions, or maybe even mid 10's of millions.
We also know specifically that Earth is twice the population of any other planet (next being Chiron in Alpha Centari), and Epsilon Leonis is almost 5 billion.  Considering Chiron was started pre-hyperdrive, let's say that it's at least 2x Epsilon Leonis' population (reasonable I think), which means Earth is 20 Billion people, leaving 65 Billion (100-20-10-5) to the other 127 (130-3) systems.  That also means more than half the total Hai population would have to be on Hai-Home to be more densely populated compared to Earth.

> 			`You send a message to Nanachi and you two meet up in the spaceport. You're still walking toward the job board, when she asks: "Back home I heard stories about how humans have a much larger population than the Hai, and that their home planet has even more people than Hai-home, is it true that there are many other human worlds with spaceports this large?" You name her a few human planets that are extremely populous, and how even some planets with fewer people than this world have incredibly large spaceports.`

Reviewer: Earth's population is 10 billion (as stated by James in the Shuttle intro).

> 			`You send a message to Nanachi and you two meet up in the spaceport. You're still walking toward the job board, when she asks: "Back home I heard stories about how humans have a much larger population than the Hai, and that their home planet has even more people than Hai-home, is it true that there are many other human worlds with spaceports this large?" You name her a few human planets that are extremely populous, and how even some planets with fewer people than this world have incredibly large spaceports.`

Author: I feel like that should be re-opened and examined (and then changed)...  Prior estimates (pre-1950s/ <https://en.wikipedia.org/wiki/Green_Revolution> ) assumed far lower population limits, not taking into account unknown technological advances that could increase the agricultural gains.  If we're able to say, turn one acre of land into 10 acres of farmland through stacking fields, then that's a potential 10x increase in food production.  Shifting to more efficient methods of food production (hydroponics, aeroponics) further increases that...

> 			`You send a message to Nanachi and you two meet up in the spaceport. You're still walking toward the job board, when she asks: "Back home I heard stories about how humans have a much larger population than the Hai, and that their home planet has even more people than Hai-home, is it true that there are many other human worlds with spaceports this large?" You name her a few human planets that are extremely populous, and how even some planets with fewer people than this world have incredibly large spaceports.`

Author: Going to resolve this as-is/no change.

Even if Earth is only 10B people, that means there are at least a few planets with high populations.  Once this PR is done, I'll look into redoing population figures across the board.

			`	While you tell her stories of many different planets you've visited, she seems to be particularly interested in Earth. "I've seen a bit of human architecture in videos of planets like Allhome, but I can't even imagine what your capital must look like. Tens of thousands of years of another species' history, it sounds even more exciting than what I heard about our landmarks in Hai-home. Captain, do you think that someday, when we no longer need to be kept hidden from humanity, that I'll be able to visit Earth, and see the beauty of your capital?" Seeing how little she knows about humanity, it's not surprising that she is unaware of the state that Earth is in.`
			choice
				`	(Avoid talking about Earth.)`
					goto avoid
				`	(Lie about Earth.)`
					goto fake

> 				`	(Tell her about how Earth actually is.)`

Reviewer: I feel like these would be more impactful as dialog. Perhaps something along the lines of the following:
```
`	"I've never actually been to Earth before, so I couldn't tell you what it's like"`
`	"Earth is a beautiful place, so it would be nice if you could see it."`
`	"Earth isn't actually that great a place to visit. Most of its inhabitants would take any change they're given to leave it."`
```
The beginning of `label avoid` would need to be changed, but `label fake` and `label truth` look like they wouldn't need to be change with because of the dialog of the choice.

Author: I feel like, if this changed, then the conversations being led into this need to be changed, and it ends up more as a detailed history lesson/lie than anything.

As of right now, I'm going to resolve this as-is/no change.  If people playing give feedback that they want it changed, I'm more than happy to look at it again.

			`	Your explanation seems to only torment Nanachi even more. "But why is your capital so full of criminals? Is human space going through a crisis that hit even your capital?"`
			choice
				`	"In some sense, the crisis has been going on since we were aware of it."`
			`	You do your best in telling her of the historical presence of crime on Earth, and of how little was done to address the root of the problem.`
			`	"I see..." She says. "I just thought it would be fun to hear about another large planet like Hai-home. Sorry for asking about it, Captain, I'm sure having to speak of it wasn't a pleasant feeling."`
			`	Although you're sure that even in Hai space there is social inequality, she probably can't fathom that the home planet of a species could be in such bad shape, considering the Hai even go as far as to send aid to the Unfettered on a regular basis.`
					goto board
			label board
			`	You two reach the job board and decide to start looking for jobs suited for Nanachi's ship.`
			choice

> 				`	"So, do you mind if I help out?  I know the job board can be confusing the first few times you check it."`

Reviewer: ```suggestion
				`	"So, do you mind if I help out? I know the job board can be confusing the first few times you check it."`
```

> 				`	"Would you like me to accompany you to the job board?"`

Reviewer: We're already here...
I get the sense that this was added to break so that this wasn't just a single choice. How about instead you say `"Do you think you have this on your own?"` for a second choice, then have Nanachi say `"Yeah, I should be good."`

> 			`	"Thanks, but I have to do this on my own. I promise I won't mess this up by getting greedy again," She approaches the board on her own, and it becomes clear that it's much easier to get a job here: she's barely started talking to one of the desk workers before receiving a long list of jobs for smaller ships. She is certainly still very tired, but this seems to have cheered her up a bit, it is the first time she can get a job without your help after all.`

Reviewer: Run-on.
```suggestion
			`	"Thanks, but I have to do this on my own. I promise I won't mess this up by getting greedy again," She approaches the board on her own, and it becomes clear that it's much easier to get a job here: she's barely started talking to one of the desk workers before receiving a long list of jobs for smaller ships. She is certainly still very tired, but this seems to have cheered her up a bit. It is the first time she can get a job without your help after all.`
```

			`	While you thought she would've picked one or two jobs quickly and headed back to her ship, she keeps going through them and double checking the information for each one, as if something about them bothers her.`
			`	"What's wrong? Are you struggling to pick between two different jobs?" you ask.`
			`	"These all seem very easy and I could even accept many at a time, but the problem is their pay isn't very good." She grabs some worn papers from her pocket, and shows it to you: they're the documents for her loan. "The bank back home would only give me enough for the ship if I agreed on a short-term loan. I lost two days before you came along to help me, and if I don't get some better paying job soon, I won't even be able to pay the interest."`

> 			`	As you're trying to figure out a way to get the most out of the available missions, you feel a tap on your shoulder, and turn to see a Hai standing beside a human couple. The couple introduce themselves as Xavier and Vanessa, who are here with their Hai friend Narancia. "We overheard your conversation and are willing to pay a little extra for a job. We're part of a group organizing a big festival in Allhome to celebrate the friendship between humans and Hai, and we just got a last-minute idea: we've already arranged for lots of human cuisine to be transported there, and we'd like to get our hands on some spices the Quarg have in the ringworld here."`

Reviewer: I feel like this last bit need shortened. 
```suggestion
			`	As you're trying to figure out a way to get the most out of the available missions, you feel a tap on your shoulder, and turn to see a Hai standing beside a human couple. The couple introduce themselves as Xavier and Vanessa, who are here with their Hai friend Narancia. "We overheard your conversation and are willing to pay a little extra for a job. We're part of a group organizing a big festival in Allhome to celebrate the friendship between humans and Hai, and we just got a last-minute idea: we'd like to get our hands on some Quarg spices."`
```

Reviewer: Looks like this never got addressed.

Author: Wonder how I missed this...

> 			`	They tell Nanachi that they won't be going along, they already have a transport to Allhome, and will stay here a bit longer to try and advertise the event.`

Reviewer: This ends rather abruptly. No "Great, I'll take the job!" from Nanachi? No "The Quarg are on `<stopovers>`, then you can bring the spices to `<destination>`."? No mention of the deadline?

				accept
	on stopover
		dialog
			`A spaceport worker contacted Nanachi shortly before you two landed and made sure the spices were ready to be loaded in her ship. You help the workers and Nanachi make sure everything is in order, and prepare for your trip to Allhome.`
	npc accompany save
		personality escort timid
		government "Hai"
		ship "Heavy Shuttle" "Acorn Chaser"
	on visit
		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system.`
	on complete
		conversation

> 			`Allhome's docking area shows much more traffic than usual, many captains here are helping workers open cargo containers full of food, clothes, books, movies and even some mounted animals from human space. You finally meet up with the festival's organizers, who seem very excited to have acquired the exotic condiments, and have invited you two to a bar in the spaceport for a drink.`

Reviewer: ```suggestion
			`Allhome's docking area shows much more traffic than usual. Many captains here are helping workers open cargo containers full of food, clothes, books, movies and even some mounted animals from human space. You finally meet up with the festival's organizers, who seem very excited to have acquired the exotic condiments, and have invited you two to a bar in the spaceport for a drink.`
```
I feel like instead of having the player have to break the narrative and click the spaceport button, this mission could just be without an `on complete` conversation, and the next mission could be marked as `landing` with the conversation here being the start of the next mission.

Author: The suggested change is done (`, m` to `. M`)

Personally, I'm not a fan of going from deadline mission to deadline mission using `landing`.  If something bad happened in the prior deadline mission, you'd want some time to go and try to correct it.  As such, I'm going to leave that as-is.

Author: Not a fan of going directly from Deadline to Deadline missions without some chance of a break in-between them.  I'm going to leave as-is.

`, m` to `. M` was changed.

			`	"They all seemed willing to give me a job, but I ended up blowing it when I asked for more money. I just really need to get out of debt so I guess I went a bit overboard." She tries to play it cool, but it's clear that she is embarrassed about losing so many potential jobs. When you reach her ship, you take a look at the prices for commodities in Hai-home. You decide to tell Nanachi to stock up on heavy metals to sell there, to help her make a bit more profit.`
				accept
	npc accompany save
		personality escort timid
		government "Hai"
		ship "Heavy Shuttle" "Acorn Chaser"
	on visit
		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for her to arrive in this star system.`
	on complete
		conversation
			`You help the family gather all their luggage and bid them farewell, but right as they disappear into a hallway Nanachi lets out a heavy sigh. "I bit off more than I could chew. Those kids kept wanting to play with the ship's controls, and one of them tried to go up the air vent in a game of hide-and-seek. I couldn't get any rest since I had to keep making sure they weren't about to dump their own bags into the void by accident."`

> 			`	"Do you want to get some sleep before trying to find another mission?" you ask, hoping she won't try to overwork herself. "Yes, I could really use that. Sorry to make you wait again captain <last>. I'll rest a bit, see if I can't find a tourist spot nearby to relax, and meet up with you the in spaceport after a while."`

Reviewer 2: The Hai got to exciting places as tourists, so I'm not sure you want a tourist spot to "relax". (Although any tourist spot is more likely to be busy then somewhere else.)
Perhaps a park is more in order.

Author: It could also mean something like a touristy restaurant, or other area you can just go mindless in.  Personally, I see this like going to some chain restaurant, ordering some comfort food, and just "hiding in plain sight"/people-watch.  Considering she's also looking for work, a touristy spot might give a job offer...

Author: Changed to be a generic spot to relax.

Author: Changed with current version.

					goto board
			label truth
			`	She looks at you in confusion, as if trying to understand what you mean, so you explain it to her. "Centuries of widespread deforestation and pollution badly damaged the ecosystem. If you ever managed to visit, you'd be better off going to museums or monuments, but even then, there are many thieves roaming the streets, and it is very dangerous to be out at night. Lots of folks born there just think about how to get out."`
			`	You keep telling her about the issues facing humanity's birthplace, and her previous eagerness to learn about the planet fades. "How could your ancestors let their own home become like this?"`
			choice
				`	"It's rather complicated, and I'm not a historian, sociologist, or economist, but I can try and explain what I do know."`

> 			`	You recite what you read in school books or saw in documentaries, telling her about how many humans since millennia ago refused to take care of the planet, and how that piled up over hundreds of years.`

Reviewer 2: Always be careful referencing current (our) events, statements you make can become untrue or you may cause some groups to take issue. While this will probably be fine do make sure you have made a conscious decision to include it.

Author: Well, even if the current events (Climate change) are reversed, there's still a ton of other issues regarding the planet's usage that will still exist, or soon exist.  It's mainly to contrast how Hai-Home is well taken care of.

Reviewer: Instead of talking about the state of the world now (as in 21st century now) and as some claim it'll be going forward, this seems like a great opportunity to shed some light on the in-game state of Earth in terms of the overpopulation, the crime, the general state of the cities, etc. stuff that's actually mentioned in game already (planet description, Shuttle intro with James, the human space wiki page) but that we could expand upon.
It does seem though that we could cut back on some of this bit of the mission; the other two choices only lead to single paragraphs, but this choice is leading to 7 paragraphs/lines and 2 choices.

Reviewer: This also seems like a great place to make comparisons to the Unfettereds' mistreatment of their own worlds.

Author: Nanachi most likely doesn't know much, if anything beyond what the landing descriptions give; I would expand upon that mission/guy who asks for the "whole of human history" to be a far better person to go into detail.  (Would be an interesting chain of missions where you collect some scientists from all the races, do some sort of "alien meet and greet" for everyone...)

Author: Nanachi isn't likely to know much about the Unfettered...  I mean, she's a brand new captain, already having issues finding jobs...  Maybe as a post-"campaign" mission chain where you take her to Earth after she's been a captain for a while, or if you talk to that guy who wanted the "history of the Humans" data.

				decline

event "Nanachi: Update"

mission "Nanachi Meeting"
	landing
	source
		government "Hai"
		not planet "Darkwaste"
	to offer
		has "event: Nanachi: Update"
		random > 30
	on offer
		conversation
			`After you leave your ship and start making your way through the docking area, you notice a Hai running towards you and waving. As they get closer, you realize that it's Nanachi, the Hai pilot who you helped out years ago.`
			`	"I was just about to go back to my crew when I saw you coming out of your ship. I know a nice snack bar here, let's go there!" Before you can even say hello, she's dragged you to the bar, and ordered some drinks for the two of you. She tells you about her last few years as a captain. She has long since sold her old ship, and now pilots a Geocoris, transporting large amounts of cargo across Hai space. You also tell her about how your life has been ever since you last met, of your adventures and how far you've come. You keep swapping stories for hours, until the bar is ready to close.`
			`	You thank her for the drinks, and get ready to say your goodbyes.`

> 			`	"Oh wait, before you go captain, let me give you something," She says, right as you're about to head to your ship. She grabs a small pendant from her pocket. "Back home in Giverstone, when I had some free time from the mining works, I took a hobby of sculpting some small stones into ship miniatures. I thought it'd be nice to make this for you in case I saw you again." She gives you the pendant, which has been sculpted into a Hai Shield Beetle. It is extremely well made, with different surface textures representing the different parts of the Shield Beetle. "It's just a good-luck charm I thought you'd like. Thanks again captain!" You two part ways, and head back to your ships.`

Reviewer 2: If you can find an artist, this might be a nice place to add a scene showing the thing.

Author: Yea, that would be really nice.  Maybe add it as a trinket if they get added.

Reviewer: I'm sure we have some artists on Discord.

Author: I've thought about it, and while it would be great, it shouldn't hold up the missions being added at this time.  (There's also stuff like, "could we do an animated scene?" I want to discuss elsewhere first)

Author: I have some other ideas for this scene, but I need to get them worked out first (animation for the scene).  None that should delay the missions themselves though.

			`	"These all seem very easy and I could even accept many at a time, but the problem is their pay isn't very good." She grabs some worn papers from her pocket, and shows it to you: they're the documents for her loan. "The bank back home would only give me enough for the ship if I agreed on a short-term loan. I lost two days before you came along to help me, and if I don't get some better paying job soon, I won't even be able to pay the interest."`
			`	As you're trying to figure out a way to get the most out of the available missions, you feel a tap on your shoulder, and turn to see a Hai standing beside a human couple. The couple introduce themselves as Xavier and Vanessa, who are here with their Hai friend Narancia. "We overheard your conversation and are willing to pay a little extra for a job. We're part of a group organizing a big festival in Allhome to celebrate the friendship between humans and Hai, and we just got a last-minute idea: we've already arranged for lots of human cuisine to be transported there, and we'd like to get our hands on some spices the Quarg have in the ringworld here."`

> 			`	Nanachi perks up during their conversation.  "I'm sure I can get the spices to <panet> within two weeks.  I won't let you down!"`

Reviewer: ```suggestion
			`	Nanachi perks up during their conversation.  "I'm sure I can get the spices to <planet> within two weeks. I won't let you down!"`
```

> 			`	They tell Nanachi that they won't be going along; they already have a transport to Allhome, and will stay here a bit longer to try and advertise the event.`

Reviewer: I feel like this last bit needs some dialog after Nanachi just accepted the job.
```suggestion
			`	"That's wonderful," Narancia says. They tell Nanachi that they won't be going along; they already have a transport to Allhome, and will stay here a bit longer to try and advertise the event. "Just come to Allhome after you've picked up the spices. Safe travels!"`
```

			`	While you tell her stories of many different planets you've visited, she seems to be particularly interested in Earth. "I've seen a bit of human architecture in videos of planets like Allhome, but I can't even imagine what your capital must look like. Tens of thousands of years of another species' history, it sounds even more exciting than what I heard about our landmarks on Hai-home. Captain, do you think that someday, when we no longer need to be kept hidden from humanity, that I'll be able to visit Earth, and see the beauty of your capital?" Seeing how little she knows about humanity, it's not surprising that she is unaware of the state that Earth is in.`
			choice
				`	(Avoid talking about Earth.)`
					goto avoid
				`	(Lie about Earth.)`
					goto fake
				`	(Tell her about how Earth actually is.)`
					goto truth
			label avoid

> 			`	You tell her that you haven't ever visited the human capital, and that you never had much interest in learning about it. "Oh, sorry about that, shouldn't have assumed you knew everything, you're a starship captain after all, not a historian, I'll just ask someone else when I get the chance.", says Nanachi.`

Reviewer: ```suggestion
			`	You tell her that you haven't ever visited the human capital, and that you never had much interest in learning about it. "Oh, sorry about that, shouldn't have assumed you knew everything, you're a starship captain after all, not a historian, I'll just ask someone else when I get the chance," says Nanachi.`
```

					decline
			`	She thanks you profusely, and promises that she'll pay you for everything as soon as she can. The organizers are a bit hesitant, but seeing as no other captain here has decided to offer them help, they agree to let the two of you go to Prime and retrieve the cargo.`
			`	Vanessa grabs ahold of Nanachi's arm before she can run off to the spaceport. "Just remember, if you can't get back here by <day>, don't worry about it; we'll understand. I'm sure we can come up with some alternative to have the festival without our main attractions."`
			`	As you're leaving the bar, Xavier pulls you aside, speaking very quietly, "<last>, I know you're just trying to help this girl, but don't get killed over this. If you run into those pirates don't try to fight them, just rush to Prime."`
				accept
	npc
		personality nemesis waiting
		government "Pirate"
		system "Betelgeuse"
		ship "Marauder Firebird (Hai)" "Nut King Call"
		ship "Marauder Splinter (Hai)" "The Nutcracker"
		ship "Marauder Quicksilver (Hai)" "Hazelnut's Bane"
		ship "Marauder Bounder (Hai)" "Rodent Hunter"
	npc accompany save
		personality escort timid

> 		government "Hai"

Reviewer 3: Since #4738 "Hai" can no longer go through the Waypoint wormhole, so this government should be changed to " Hai "

Author: Just changed it.  Should work fine.
